page/javascript-101/types.md

```diff
@@ -66,7 +66,7 @@ var bar2;
 
 ## Objects
 
-Everything else in JavaScript is considered an object. While there are [numerous built-in objects](https://developer.mozilla.org/en/JavaScript/Reference#Global_Objects, "MDN - Global Object Reference"), this chapter will cover:
+Everything else in JavaScript is considered an object. While there are [numerous built-in objects](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects "MDN - Standard built-in objects"), this chapter will cover:
```

Comment:
`"MDN - Global Object Reference"` seems better than `"MDN - Standard built-in objects"` to me. (that's what was in originally)